packages/documentation-site/patternfly-docs/content/demos/Compass/CompassIntegrationsDemo.tsx

```diff
@@ -0,0 +1,825 @@
+              isCompact
+              variant="outline"
+            >
+              {statusText}
+            </Label>
+          ),
+        },
+        {
+          cell: <ActionsColumn items={rowActions} />,
+          props: { isActionCell: true },
+        },
+      ],
+    })
+  );
+
+  const columns = [
+    {
+      cell: undefined,
+      props: {
+        select: {
+          onSelect: () => { },
+          isSelected: false,
+          'aria-label': "Select all",
+        },
+      },
+    },
+    "Name",
+    "Type",
+    "Status",
+    {
+      cell: undefined,
+      props: {
+        screenReaderText: "Actions",
+      },
+    }
+  ];
+
+  const tableViewContent = (
+    <>
+      <DataViewToolbar
+        clearAllFilters={() => { }}
+        filters={
+          <DataViewTextFilter
+            filterId="name"
+            title="Name"
+            placeholder="Filter by name"
+          />
+        }
+        actions={
+          <ToggleGroup>
+            <ToggleGroupItem
+              icon={<ThIcon />}
+              aria-label="grid icon button"
+              isSelected={activeDisplay === "card"}
+              onChange={() => setActiveDisplay("card")}
+            ></ToggleGroupItem>
+            <ToggleGroupItem
+              icon={<ListIcon />}
+              aria-label="list icon button"
+              isSelected={activeDisplay === "table"}
+              onChange={() => setActiveDisplay("table")}
+            ></ToggleGroupItem>
+          </ToggleGroup>
+        }
+        pagination={<Pagination page={1} perPage={10} isCompact />}
+      />
+      <DataViewTable
+        aria-label="Integrations"
+        columns={columns}
+        rows={rows}
+        className="pf-m-plain"
+      />
+    </>
+  );
+
+  const mainContent = (
+    <>
+      <CompassMainHeader title={<Title headingLevel="h1">Integrations</Title>} toolbar={(
+        <Toolbar hasNoPadding>
+          <ToolbarContent>
+            <ToolbarGroup>
+              <ToolbarItem>
+                <Button
+                  variant="primary"
+                  onClick={onDrawerToggle}
+                >
+                  Toggle drawer
+                </Button>
+              </ToolbarItem>
+            </ToolbarGroup>
+            <ToolbarGroup>
+              <ToolbarItem>
+                <Button icon={<EllipsisVIcon />} variant="plain" aria-label="More options" />
+              </ToolbarItem>
+            </ToolbarGroup>
+          </ToolbarContent>
+        </Toolbar>
+      )} />
+      <CompassContent>
+        <CompassPanel isScrollable id="integrations-main-content" tabIndex={-1}>
+          {(() => {
+            if (activeDisplay === "table") {
+              return tableViewContent;
+            } else {
+              return cardViewContent;
+            }
+          })()}
+        </CompassPanel>
+      </CompassContent>
+    </>
+  );
+  const footerContent = (
+    <CompassMessageBar>
+      <CompassPanel isPill hasNoPadding>
```

Comment:
This shouldn't have a border

```suggestion
      <CompassPanel isPill hasNoPadding hasNoBorder>
```

Goes from this

<img width="490" height="103" alt="Image" src="https://github.com/user-attachments/assets/f69283a1-6fb2-4706-a648-8a2da895b547" />

to this

<img width="482" height="86" alt="Image" src="https://github.com/user-attachments/assets/98942e99-2cba-418e-a0e3-14d0b7012a28" />